app/src/main/java/com/example/sony/training/MainActivity.java

```diff
@@ -2,12 +2,54 @@
 
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.view.View;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Spinner;
+import android.widget.TextView;
+import android.widget.Toast;
 
-public class MainActivity extends AppCompatActivity {
+import org.xml.sax.Parser;
+
+public class MainActivity extends AppCompatActivity implements View.OnClickListener {
+
+    private Spinner mSpinner;
+    private Button mCalculate;
+    private TextView mTotal;
+    private EditText mBill;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_main);
+        setContentView(R.layout.layout);
+
+        //widget
+        mCalculate = (Button) findViewById(R.id.btnCalculate);
+        mSpinner = (Spinner) findViewById(R.id.Spn);
+        mTotal = (TextView) findViewById(R.id.txtTotal1);
+        mBill = (EditText) findViewById(R.id.edtBill);
+
+        //Spinner
+        ArrayAdapter<CharSequence> adapter = ArrayAdapter.createFromResource(this,
+                R.array.spinner1, android.R.layout.simple_spinner_item);
+        adapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
+        mSpinner.setAdapter(adapter);
```

Comment:
pls define method ``initViews`` and move this code (line 27 -> 37) into this method